app/views/users/show.html.erb

```diff
@@ -5,53 +5,52 @@
 
   links = {}
   [
-    [ :comments,                      :comment,      :show_comments_by_user           ],
-    [ :comments_for,                  :comment,      :show_comments_for_user          ],
-    [ :images,                        :image,        :images_by_user                  ],
-    [ :location_descriptions_authors, :location,     :location_descriptions_by_author ],
-    [ :location_descriptions_editors, :location,     :location_descriptions_by_editor ],
-    [ :locations,                     :location,     :locations_by_user               ],
-    [ :locations_versions,            :location,     :locations_by_editor             ],
-    [ :name_descriptions_authors,     :name,         :name_descriptions_by_author     ],
-    [ :name_descriptions_editors,     :name,         :name_descriptions_by_editor     ],
-    [ :names,                         :name,         :names_by_user                   ],
-    [ :names_versions,                :name,         :names_by_editor                 ],
-    [ :observations,                  :observer,     :observations_by_user            ],
-    [ :species_lists,                 :species_list, :species_lists_by_user           ],
-    [ :life_list,                     :observer,     :checklist                       ]
+    [ :comments, :comment, :show_comments_by_user ],
+    [ :comments_for, :comment, :show_comments_for_user ],
+    [ :images, :image, :images_by_user ],
+    [ :location_description_authors, :location,
+      :location_descriptions_by_author ],
+    [ :location_description_editors, :location,
+      :location_descriptions_by_editor ],
+    [ :locations, :location, :locations_by_user ],
+    [ :locations_versions, :location, :locations_by_editor ],
+    [ :name_description_authors, :name, :name_descriptions_by_author ],
+    [ :name_description_editors, :name, :name_descriptions_by_editor ],
+    [ :names, :name, :names_by_user ],
+    [ :names_versions, :name, :names_by_editor ],
+    [ :observations, :observer, :observations_by_user ],
+    [ :species_lists, :species_list, :species_lists_by_user ],
+    [ :life_list, :checklists, :show ]
```

Comment:
You don't like lined up table columns?  Or is this rubocop's preference?

Reply:
Lines were too long, so it was a train wreck in my editor. (It forces line-wraps)

Reply:
I thought it might be something like that.  No worries.